rootfs/etc/cont-init.d/08-svc-snmptrapd.sh

```diff
@@ -2,6 +2,15 @@
 
 SIDECAR_SNNMPTRAPD=${SIDECAR_SNNMPTRAPD:-0}
 LIBRENMS_SNMP_COMMUNITY=${LIBRENMS_SNMP_COMMUNITY:-librenmsdocker}
+SNMP_ACTION=${SNMP_ACTION:-log,execute,net}
+SNMPV3_USER=${SNMPV3_USER:-librenms_user}
+SNMPV3_AUTH=${SNMPV3_AUTH:-auth_pass}
+SNMPV3_PRIV=${SNMPV3_PRIV:-priv_pass}
+SNMPV3_AUTH_PROTO=${SNMPV3_AUTH_PROTO:-SHA}
+SNMPV3_PRIV_PROTO=${SNMPV3_PRIV_PROTO:-AES}
+SNMPV3_SECURITY_LEVEL=${SNMPV3_SECURITY_LEVEL:-priv}
```

Comment:
I think we should prefix with `SNMP_` instead of `SNMPV3_`.

Reply:
This suggestion has been applied